src/services/export.test.js

```diff
@@ -43,30 +59,95 @@ const initBrowser = async () => {
 const initPuppeteer = async () => {
   await initBrowser();
   page = await browser.newPage();
-  await page.goto(`file://${__dirname}/export.test.html`);
+  await page.goto(`file://${__dirname}/export.test.html`, {
+    waitUntil: 'domcontentloaded',
+    timeout: 0,
+  });
 };
 
 const initPuppeteerWithMode = async () => {
   await initBrowser();
   pageStatic = await browser.newPage();
-  await pageStatic.goto(`file://${__dirname}/export_static.test.html`);
+  await pageStatic.goto(`file://${__dirname}/export_static.test.html`, {
+    waitUntil: 'domcontentloaded',
+    timeout: 0,
+  });
   pageReadOnly = await browser.newPage();
-  await pageReadOnly.goto(`file://${__dirname}/export_readonly.test.html`);
+  await pageReadOnly.goto(`file://${__dirname}/export_readonly.test.html`, {
+    waitUntil: 'domcontentloaded',
+    timeout: 0,
+  });
   pageInteractive = await browser.newPage();
   await pageInteractive.goto(
-    `file://${__dirname}/export_interactive.test.html`
+    `file://${__dirname}/export_interactive.test.html`,
+    { waitUntil: 'domcontentloaded', timeout: 0 }
   );
 };
 
 const closePuppeteer = async () => {
   await browser.close();
 };
 
+const removeAllImages = async () => {
+  const directory = TMP_FOLDER;
+  fs.readdir(directory, (err, files) => {
+    if (err) throw err;
```

Comment:
For the `if` clauses, it's better if you wrap them in `{ }` as it avoids typos later and we can test the coverage of each branch easily..